lib/datasource/rubygems/get.ts

```diff
@@ -29,12 +32,95 @@ export async function fetch(
   return response.body;
 }
 
+export async function fetchDependencies(
+  dependencies: string[],
```

Comment:
Why multiple? I think we always fetch only one dependency

Reply:
> Why multiple? I think we always fetch only one dependency

Because the API accepts multiple dependencies. Should I limit it to one dependency for now?

Reply:
Yes, limit to one dep to simplify our api.

Reply:
90056f3